test/unit/components/monitor.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import DefaultMonitor from '../../../src/components/monitor/default-monitor';
+import Monitor from '../../../src/components/monitor/monitor';
+
+jest.mock('../../../src/lib/themes/default-colors');
+
+describe('Monitor Component', () => {
```

Comment:
It could make sense to add at least one more test of a different theme

Reply:
I like both of these ideas. Pushed some changes. I don't feel these new changes need to be reviewed. But if you do, you should be aware that I rebase and squashed this PR into a single commit. That makes pulling the changes if you've already pulled them a bit tricky.